src/script_lua.c

```diff
@@ -1455,6 +1474,14 @@ void luaRegisterRedisAPI(lua_State* lua) {
     lua_pushcfunction(lua,luaRedisSetReplCommand);
     lua_settable(lua,-3);
 
+    lua_pushstring(lua,"elapsed_start");
+    lua_pushcfunction(lua,redis_elapsed_start);
+    lua_settable(lua,-3);
+
+    lua_pushstring(lua,"elapsed_us");
+    lua_pushcfunction(lua,redis_elapsed_us);
+    lua_settable(lua,-3);
```

Comment:
i'm not certain we need these.
in any case, we don't expect many to use the features added by this PR, they'll only be used by rare and complicated scripts.
in which case, since we expose the lua system APIs (os.clock) anyway, i don't think we need to expose any convenience wrappers.

it's true that the os.clock approach is more easily abused, and in these we guide people towards something safer. but since we expose it anyway, and since this is reserved to very few specific cases, i don't tihink we need to bother ourselves with inventing any specific API for them, and we can suffice by exposing the built-in system ones.

@madolson i realize you were driving towards that direction, so please respond.

Reply:
These api's provide microsecond granularity, which I think is pretty important since most of these scripts are going to execute at that scale. `os.clock` returns only to a second granularity (or I was assuming based on the top comment and documentation). I was less clear about the value of `os.clock`.

Reply:
you're right. `os.time()` is in seconds and has no fractional part.
`os.clock()` measures CPU usage (i assume using `getrusage`), and it indeed returns seconds, but has fractional part.
so if the user wants to benchmark or limit the processing time of his script, `os.clock()` could be sufficient.
if the user wants to know the wall-clock time, they have `os.time()` with no precision.
if they want to measure a monotonic time that's not tied to the CPU used, they need the `elapsed_start` API added here.

the issue that led to this PR said:
>I use the previous behavior to store the result of queries that take a long time to execute

for that, i suppose `os.clock()` is sufficient.
WDYT?

Reply:
> for that, i suppose os.clock() is sufficient.

That is probably true. The original ask could probably be satisified with just `os.clock()`.

The other perspective I had is to encourage folks to use the right value, the start + elapsed paradigm makes it hard to misuse. Someone could possibly think os.clock() is from the start of the current evaluation during testing, since it's not that clearly defined.

I have a weak preference for the current set of APIs, but could be con

Reply:
since the use case for this is really rare, i prefer not to create any new APIs if we can suffice by just exposing a built in one.